bin/suitcss

```diff
@@ -65,7 +65,13 @@ program.parse(process.argv);
 
 var input = program.args[0] ? resolve(program.args[0]) : null;
 var output = program.args[1] ? resolve(program.args[1]) : null;
-var config = program.config ? require(resolve(program.config)) : null;
+var config;
+var configFile = program.config;
+if (configFile && configFile.endsWith('.js')) {
+  config = require(resolve(configFile));
+} else if (configFile) {
+  config = fs.readJsonSync(resolve(configFile));
+}
```

Comment:
ah, I went too fast & pushed an factor out commit already, sry. 
however, maybe once the importRoot stuff also gets more code in the cli, it might get a bit cluttered?
anyway, just ping me if you'd like this changed!